corehq/apps/geospatial/views.py

```diff
@@ -275,10 +274,7 @@ def _get_paginated_users_without_gps(domain, page, limit, query):
         UserES()
         .domain(domain)
         .mobile_users()
-        .OR(
-            missing_metadata_property(location_prop_name),
-            missing_metadata_value(location_prop_name)
-        )
+        .missing_or_empty_metadata_property(location_prop_name)
```

Comment:
Nice! thank you.

You could also now get rid of the individual functions, if that seems better, since they are small and not really used elsewhere.

Reply:
I'm tempted to keep them to better document what each one is doing, however these could be made into private functions so that externally we only use `missing_or_empty_metadata_property`.

Reply:
[55461a1](https://github.com/dimagi/commcare-hq/pull/33420/commits/55461a1c442aa74a1dbae565a9dd89e7dc4456c2)

Reply:
That is even better than removing, thanks!